packages/graphql-playground-react/src/components/Playground/DocExplorer/FieldDoc.tsx

```diff
@@ -53,6 +58,13 @@ export default class FieldDoc extends React.Component<Props, State> {
     const implementationsOffset =
       obj.fields.length + obj.interfaces.length + obj.args.length
 
+    const typeInstance =
```

Comment:
I am not happy this style of coding but was trying to follow lint rules. I can change it to something like this:
``` js
if (type instanceof GraphQLInterfaceType) {
  typeInstance = 'interface'
} else if (type instanceof GraphQLUnionType) {
  typeInstance = 'union'
} else if (type instanceof GraphQLEnumType) {
  typeInstance = 'enum'
} else {
  typeInstance = 'type'
}
```
What looks better in my eyes

Reply:
Yes, that look definitely better! You can also identify that linting rule and ignore it in the tslint config.
I already had a lot of cases where this rule forced you to do this ugly ternary stuff.